exec/vector/src/main/codegen/templates/VariableLengthVectors.java

```diff
@@ -506,6 +506,8 @@ public boolean isNull(int index){
   }
 
   /**
+   * <h2>Overview</h2>
```

Comment:
Nit, but I think that h4 is the usual heading level used in Java doc comments. The higher levels are used in the surrounding generated HTML.

Reply:
Fixed